src/plugins/intel_cpu/src/nodes/convert.cpp

```diff
@@ -144,15 +152,27 @@ void Convert::execute(dnnl::stream strm) {
     if (parentPaddElemCount != childPaddElemCount)
         IE_THROW() << errorPrefix << " has different elements number in input and output buffers";
 
-    void* srcPtr = parentMem.GetPtr();
-    void* dstPtr = childMem.GetPtr();
+    convertParams.size = parentPaddElemCount;
+
+    std::vector<MemoryCPtr> srcMemory;
+    for (size_t i = 0; i < getOriginalInputsNumber(); i++) {
+        srcMemory.push_back(getParentEdgeAt(i)->getMemoryPtr());
+    }
+    std::vector<MemoryPtr> dstMemory;
+    for (size_t i = 0; i < getOriginalOutputsNumber(); i++) {
+        dstMemory.push_back(getChildEdgeAt(i)->getMemoryPtr());
+    }
 
-    cpu_convert(srcPtr,
-                dstPtr,
-                parentMem.getDesc().getPrecision(),
-                origPrc,
-                childMem.getDesc().getPrecision(),
-                parentPaddElemCount);
+    dnnl::primitive_attr attr;
+    auto selectedPD = getSelectedPrimitiveDescriptor();
+    std::vector<MemoryDescPtr> srcDescs = {parentMem.getDescPtr()};
+    std::vector<MemoryDescPtr> dstDescs = {childMem.getDescPtr()};
+    execPtr = selectedPD->getExecutorFactoryAs<ConvertExecutorFactory>()->makeExecutor(convertParams,
```

Comment:
Could you please create a prepareParams method for the convert node to align with the rest of the nodes.

Reply:
fixed